.flake8

```diff
@@ -328,3 +326,10 @@ rst-roles =
 
 # wemake-python-styleguide
 show-source = true
+
+# flake8-quotes (pulled in by wemake style guide)
+# https://github.com/zheller/flake8-quotes
+# at bottom of configuration because some IDEs
```

Comment:
Also, it's probably worth specifying that because of `inline-quotes`, the set of settings should remain at the bottom of the whole file. There's already a pattern of keeping any settings contributed by third-party plugins below flake8 "stdlib" options that I've adopted long ago for clarity.

Reply:
it was clear to me :)  I'll rephrase to be more specifc

Reply:
resolving after approved and implemented